core/jazz_create-serverless-service/index.js

```diff
@@ -208,29 +208,36 @@ var getServiceData = (service_creation_data, authToken, configData) => {
             "DESCRIPTION": service_creation_data.description,
             "TYPE": service_creation_data.service_type,
             "RUNTIME": service_creation_data.runtime,
-            "REGION": service_creation_data.region,
             "USERNAME": user_id,
             "IS_PUBLIC_ENDPOINT": service_creation_data.is_public_endpoint || false,
             "STATUS": "creation_started"
         };
 
         var serviceMetadataObj = {};
         if (service_creation_data.tags) {
-            inputs.TAGS = service_creation_data.tags;
+          inputs.TAGS = service_creation_data.tags;
         }
 
         if (service_creation_data.email) {
-            inputs.EMAIL = service_creation_data.email;
+          inputs.EMAIL = service_creation_data.email;
         }
 
         if (service_creation_data.slack_channel) {
-            inputs.SLACKCHANNEL = service_creation_data.slack_channel;
+          inputs.SLACKCHANNEL = service_creation_data.slack_channel;
         }
 
         if ((service_creation_data.service_type === "api" || service_creation_data.service_type === "function") && (service_creation_data.require_internal_access)) {
-            serviceMetadataObj.require_internal_access = service_creation_data.require_internal_access;
+          serviceMetadataObj.require_internal_access = service_creation_data.require_internal_access;
         }
 
+        let accountId = (service_creation_data.accountId?service_creation_data.accountId:configData.ACCOUNT_ID)
+        let provider = (service_creation_data.provider?service_creation_data.provider:configData.PROVIDER)
+        let region = (service_creation_data.region?service_creation_data.region:configData.REGION)
+
+        serviceMetadataObj.accountId = accountId.split(",");
```

Comment:
what is the format of this data? why are you doing split here?

Reply:
I am not sure about the format of the data coming from ui now. I am expecting a comma separated string. That's why splitting by comma.

Reply:
@devsatishm @chimpuz is this correct? Is UI going to be a comma-separated list. Something is not right here

Reply:
Checked with @rjsughosh, as per him it is coming as array from ui. So removing the splitting code.